src/_cffi_src/openssl/ssl.py

```diff
@@ -255,6 +255,26 @@
                                          unsigned char *,
                                          unsigned int
                                      ));
+void SSL_CTX_set_psk_find_session_callback(SSL_CTX *,
+                                           int (*)(
+                                               SSL *,
+                                               const unsigned char *,
+                                               size_t,
+                                               SSL_SESSION **
+                                           ));
+void SSL_CTX_set_psk_use_session_callback(SSL_CTX *,
+                                          int (*)(
+                                              SSL *,
+                                              const EVP_MD *,
+                                              const unsigned char **,
+                                              size_t *,
+                                              SSL_SESSION **
+                                          ));
+const SSL_CIPHER *SSL_CIPHER_find(SSL *, const unsigned char *);
+SSL_SESSION *SSL_SESSION_new(void);
+int SSL_SESSION_set1_master_key(SSL_SESSION *, const unsigned char *,
```

Comment:
This isn't a valid function declaration.

We also are compatible with several versions of libressl as well as the latest boringssl main branch commit, so any changes need to pass CI with them as well (which we can see once the syntax issue is resolved).

Is the intention to add TLS 1.3 PSK to pyOpenSSL?